AsyncDisplayKit/Layout/ASLayout.mm

```diff
@@ -47,6 +47,11 @@ @interface ASLayout () <ASDescriptionProvider>
  */
 @property (nonatomic, getter=isFlattened) BOOL flattened;
 
+/**
+ * Array for explicitly retain sublayout layout elements in case they are created and references in layoutSpecThatFits: and no one else will hold a strong reference on it
+ */
+@property (nonatomic, strong) NSMutableArray *sublayoutLayoutElements;
```

Comment:
@maicki what do you think about making this private / an internal ivar?

Should we do the optimization to check layoutElementType, since it's quick?